Dialog/CreditEditor_TableView.h

```diff
@@ -1,15 +1,15 @@
-﻿#ifndef CUSTOMQTABLEVIEW_H
-#define CUSTOMQTABLEVIEW_H
+﻿#ifndef CREDITSEDITOR_TABLEVIEW_H
```

Comment:
The symbol still does not match, it should be CREDITEDITOR_TABLEVIEW_H

Reply:
Or perhaps the header file itself should be CreditsEditor_TableView.h . Change whichever you think is more appropriate

Reply:
:heavy_check_mark: